bindings/girlmath-js/src/lib.rs

```diff
@@ -15,4 +9,110 @@ pub mod exchange;
 #[wasm_bindgen(start)]
 pub fn start() {
     console_error_panic_hook::set_once();
-}
\ No newline at end of file
+}
+
+/// Converts a major currency unit (like dollars) to a minor currency unit (like cents).
+///
+/// # Arguments
+/// * `amt` - A string slice that holds the amount in major currency units.
+/// * `currency_code` - The ISO or cryptocurrency code as a string slice.
+///
+/// # Returns
+/// A `Result<String, String>` which is Ok with the amount in minor units
+/// or an Err with an error message.
+///
+/// # Examples
+/// ```
+/// use girlmath::major_to_minor;
+///
+/// let cents = major_to_minor("10.50", "USD").unwrap();
+/// assert_eq!("1050", cents);
+/// ```
+#[wasm_bindgen(js_name = majorToMinor)]
+pub fn major_to_minor_wasm(amt: &str, currency_code: &str) -> Result<String, JsError> {
+    return match girlmath::major_to_minor(amt, currency_code) {
+        Ok(val) => Ok(val),
+        Err(e) => Err(JsError::new(&e)),
+    };
+}
+
+/// Converts a minor currency unit (like cents) to a major currency unit (like dollars).
+///
+/// # Arguments
+/// * `amt` - A string slice that holds the amount in minor currency units.
+/// * `currency_code` - The ISO or cryptocurrency code as a string slice.
+///
+/// # Returns
+/// A `Result<String, String>` which is Ok with the amount in major units
+/// or an Err with an error message.
+///
+/// # Examples
+/// ```
+/// use girlmath::minor_to_major;
+///
+/// let dollars = minor_to_major("1050", "USD").unwrap();
+/// assert_eq!("10.50", dollars);
+/// ```
+#[wasm_bindgen(js_name = minorToMajor)]
+pub fn minor_to_major_wasm(amt: &str, currency_code: &str) -> Result<String, JsError> {
+    return match girlmath::minor_to_major(amt, currency_code) {
+        Ok(val) => Ok(val),
+        Err(e) => Err(JsError::new(&e)),
+    };
+}
+
+/// Converts an amount from one currency to another using a given exchange rate.
+///
+/// # Arguments
+/// * `payin_amt` - The amount in the pay-in currency as a string slice.
+/// * `payin_currency` - The pay-in currency code as a string slice (currently unused).
+/// * `payout_currency` - The pay-out currency code as a string slice.
+/// * `rate` - The exchange rate as a string slice.
+///
+/// # Returns
+/// A `Result<String, String>` which is Ok with the converted amount
+/// or an Err with an error message.
+///
+/// # Examples
+/// ```
+/// use girlmath::convert;
+///
+/// let payout_amt = convert("3", "USD", "GHS", "12.10").unwrap();
+/// assert_eq!("36.30", payout_amt);
+/// ```
+#[wasm_bindgen(js_name = convert)]
+pub fn convert_wasm(
+    payin_amt: &str,
+    _payin_currency: &str,
+    payout_currency: &str,
+    rate: &str,
+) -> Result<String, JsError> {
+    return match girlmath::convert(payin_amt, _payin_currency, payout_currency, rate) {
+        Ok(val) => Ok(val),
+        Err(e) => Err(JsError::new(&e)),
+    };
```

Comment:
Slightly simpler way to do this in rust.
```suggestion
    girlmath::convert(payin_amt, _payin_currency, payout_currency, rate)
        .map_err(|e| JsError::new(&e))
```

Does wrapping in `JsError` do something nide for DevX on the JS side?

Reply:
doneski https://github.com/TBD54566975/girlmath/pull/6/commits/3b45eb7f14de983b35eb594597ecc30dad1bb98b